File: src/main/webapp/app/model/JobSeekerDto.ts
export class JobSeekerDto {
  id: number | undefined
  firstName: string | undefined
  lastName: string | undefined
  phoneNumber: string | undefined
  status: string | undefined
  gender: string | undefined

  constructor(id: number, firstName: string, lastName: string, phoneNumber: string, status: string, gender: string) {
    this.id = id
    this.firstName = firstName
    this.lastName = lastName
    this.phoneNumber = phoneNumber
    this.status = status
    this.gender = gender
  }
}

File: src/main/kotlin/fr/audithor/dto/jobseeker/JobSeekerDto.kt
package fr.audithor.dto.jobseeker

import model.Gender
import model.JobSeekerStatus

data class JobSeekerDto(
  val id: Long,
  val firstName: String,
  val lastName: String,
  val phoneNumber: String,
  var status: JobSeekerStatus,
  var gender: Gender
)
